external/hiring.md

```diff
@@ -0,0 +1,40 @@
+# Software Developer Hiring Process
+
+## Purpose
+
+To standardize and organize our hiring process for tech team devs, so we can
+be more inclusive and transparent to candidates.
+
+## Steps
+
+1. Resume review
+    - You can expect several team members to review your resume
+    - Don't stress too much about this, we know getting to know you takes more than a resume.
+2. Phone screen candidate
+    - ~25 minutes with one **Tech Team** member
+    - We'll give you a high level description of PRX, team and role
+    - You'll get a chance to tell us about your background and experience
+3. Technical venture
+    - We'll work with you to pick something that best highlights your work and experience
+    - Could be:
+      - Async code challenge
+      - An OSS pull request
+      - Something they've already built
+      - ...
+    - To be respectful of your time we're trying to get each candidate to spend no more than 2hrs on this task
+    - You should be prepared to run/edit your project at the interview
```

Comment:
Is the expectation that candidates will need to edit their project on the fly? 

Is walkthrough a more appropriate term?

Reply:
Yeah, maybe something closer to "you should be prepared to run, walk us through, and discuss your project"?

May also want to make it clear that this gets turned in to us before (4) happens.  So we have time to get familiar with it, decide which team member is best suited to review, and formulate some questions/discussion points?

Also wondering if `4. Technical Interview` should have this bullet?  Since that's where this will actually occur.